### DISCUSSION TITLE
Doesn't work on Monterey 12.0.1 for alpha network adapter awus036h

### DISCUSSION THREAD

**Discussion by musali011** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#149] [General]
Just installed the app on my macOS Monterey 12.0.1, and followed all the steps, but didn't work as you see in the screenshot!!!

![for Chris1111](https://user-images.githubusercontent.com/94232078/141648120-d0340705-4085-4b7a-9d52-0cbe43be066b.png)

This is Alfa network adapter

![$(KGrHqNHJBUFH)7,0ykIBSEsmWiSbw~~60-1](https://user-images.githubusercontent.com/94232078/141648144-decd4ff7-2a75-4068-849f-71346e9533f0.JPG)

**Comment by chris1111** [comment]
Its an OLD adapter support only from [macOS 10.4 and 10.5](https://files.alfa.com.tw/?dir=%5B1%5D%20WiFi%20USB%20adapter/AWUS036H)

**Comment by suculent** [comment]
According to ALFA, this is issue caused by Realtek not providing drivers for recent MacOS (both for Intel/Silicon).

**Comment by MuhammadAbdullah54321** [comment]
Is there some work around to make it work on latest Mac OS versions like Ventura
